Prox/Prox/Utilities/PlaceUtilities.swift

```diff
@@ -56,8 +56,10 @@ struct PlaceUtilities {
         }
     }
 
-    private static func getTravelTimes(forPlaces places: [Place], fromLocation location: CLLocation, withTransitTypes transitTypes: [MKDirectionsTransportType]) -> Future<[DatabaseResult<TravelTimes>]> {
-        return places.map { $0.travelTimes(fromLocation: location, withTransitTypes: transitTypes) }.allFilled()
+    private static func updateTravelTimes(forPlaces places: [Place], fromLocation location: CLLocation, withTransitTypes transitTypes: [MKDirectionsTransportType]) -> Future<Void> {
+        // HACK: travelTimes(fromLocation: location) may not fill its Deferred due to an apparent bug where
+        // MKDirections.calculateETA() never executes its callback. We set a timeout as a workaround.
+        return places.map { $0.travelTimes(fromLocation: location, withTransitTypes: transitTypes) }.allFilled().timeout(deadline: .now() + 5).ignored()
```

Comment:
The `places.map { ... }.allFilled().timeout(...)` will return a `Future<[DatabaseResult<TravelTimes>]?>`, so the `ignored()` converts this to `Future<Void>` to match the function return type.

I changed `getTravelTimes` -> `updateTravelTimes`, where we now return a `Future<Void>` instead of `Future<[DatabaseResult<TravelTimes>]>`. I did that because 1) we weren't using the result and 2) because the method also updates each `Place`, and side effects for getters are bad.